src/ruby/spec/generic/rpc_server_spec.rb

```diff
@@ -755,5 +757,59 @@ def common_check_of_finished_server_call(call)
         check_multi_req_view_of_finished_call(@service.server_side_call)
       end
     end
+
+    if Object.const_defined?('UNIXSocket')
```

Comment:
I believe we can actually remove this conditional. There is a large TODO for this but continuous jenkins tests are actually not running windows/mingw builds for grpc-ruby currently - I think we can add platform compatibility for this test in once that becomes the case, as I believe this should pass on mac and linux.